src/main/java/edu/cornell/kfs/sys/web/service/impl/CuApiJsonWebRequestReaderImpl.java

```diff
@@ -0,0 +1,33 @@
+package edu.cornell.kfs.sys.web.service.impl;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import edu.cornell.kfs.sys.web.service.CuApiJsonWebRequestReader;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.ws.rs.BadRequestException;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+
+public class CuApiJsonWebRequestReaderImpl implements CuApiJsonWebRequestReader {
```

Comment:
Once the review is passed I plan to create a separate user story to use this method in the CapAsset API.

Reply:
sounds like a good plan. Will that require full regression testing of the CapAsset tool?